tests/unit/routes/rentals/index-test.js

```diff
@@ -0,0 +1,22 @@
+import { moduleFor, test } from 'ember-qunit';
+import startMirage from '../../../helpers/setup-mirage-for-unit-test';
+import Ember from 'ember';
+
+moduleFor('route:rentals/index', 'Unit | Route | rentals/index', {
+  needs: ['model:rental'],
+  beforeEach() {
+    startMirage(this.container);
```

Comment:
Since we are already using mirage for persisted data, might as well use it for http request mocking in unit tests.  We can go this way or just stub the store.  I preferred this since its actually testing the validity of our ember data usage.